pkg/meta/meta.go

```diff
@@ -55,57 +52,26 @@ func processDrop(ctx context.Context, dstmt spqrparser.Statement, isCascade bool
 			return cli.DropKeyRange(ctx, []string{stmt.KeyRangeID})
 		}
 	case *spqrparser.ShardingRuleSelector:
-		if stmt.ID == "*" {
-			if rules, err := mngr.DropShardingRuleAll(ctx); err != nil {
-				return cli.ReportError(err)
-			} else {
-				return cli.DropShardingRule(ctx, func() string {
-					var ret []string
-
-					for _, rule := range rules {
-						ret = append(ret, rule.ID())
-					}
-
-					return strings.Join(ret, ",")
-				}())
-			}
-		} else {
-			spqrlog.Zero.Debug().Str("rule", stmt.ID).Msg("parsed drop")
-			err := mngr.DropShardingRule(ctx, stmt.ID)
-			if err != nil {
-				return cli.ReportError(err)
-			}
-			return cli.DropShardingRule(ctx, stmt.ID)
-		}
+		return cli.ReportError(spqrerror.New(spqrerror.SPQR_INVALID_REQUEST, "sharding rules are removed from SPQR"))
```

Comment:
Maybe just create a well-known error in spqrerror package? For code-reuse purposes